source/blog/2015-10-21-docker-brno-meetup-2.md

```diff
@@ -0,0 +1,57 @@
+
+The current UX of choice for muli-container applications is the README or the even more scary `curl http://really.not.dangerous.com/install.sh | bash` No one likes these options as is evidenced by everyone packaging their own version of tools.  For example, there are 454 MariaDB images on DockerHub right now.
+
+The Nulecule specification provides a way to specify all of the images that are required to run an application and provides for their discovery.  It also cleanly defines how the containers interelate and what parameters, storage, etc. they require.  It also allows them to be operated on as a group and to be handed off to various orchestration providers easily, even though the orchestrators currently all have unique and mostly incompatible format for their specifications.  Atomic App is the reference implementation of the Nulecule spec, and with the '/usr/bin/atomic' helper command can reduce a multi-container application to a single line `atomic run application` or without the helper, a single long docker command.
+
+The specification is open and not dependent on a specific container technology or orchestrator.  It allows for easy tweaking of application meta-data and parameters when moving between environments, i.e. from DEV to TEST to PRODUCTION.
+
+In a follow-up demo, Vaclav showed off the standard [guestbook-go example](https://github.com/projectatomic/nulecule/tree/master/examples/guestbook-go) and a single line installation of [Gitlab](https://github.com/navidshaikh/nulecule/tree/fix-160/examples/gitlab-centos7-atomicapp).
+
+[Yury Tsarev](https://cz.linkedin.com/in/yurytsarev) from GoodData presented, ["Test Driven Infrastrucure with Docker, Test Kitchen and Serverspec"](https://drive.google.com/a/redhat.com/file/d/0B5OHcgvKZLcdcFJkbGZVQkZvTnM/view)  Yury strongly believes that infrastructure code should be treated like any other code.  This means apply a test driven development model, storing it in a source control system and building a regression test suite.  He suggests doing this with [Test Kitchen](kitchen.ci), a pluggable and extensible test orchestrator that originated in the Chef community.  Using Test Kitchen's [docker provider](github.  com/portertech/kitchen-docker), a docker container can be used to simulate a machine under test.  Then [Serverspec](serverspec.org) can verify that the configuration code, Puppet in Yury's case, properly setup the machine.  Shell mocking is used to bypass external dependencies and docker limitations.
```

Comment:
Hi, there are some broken embedded links in this chunk
```
s#kitchen.ci#http://kitchen.ci#
s#github.  com/portertech/kitchen-docker#http://github.com/portertech/kitchen-docker#
s#serverspec.org#http://serverspec.org#
```
Thanks for a very nice writing